### DISCUSSION TITLE
Add Payment Gateway indonesia Xendit

### DISCUSSION THREAD

**Discussion by rendrandy** [inspireui/support#5472] [Request new Native Payment]
can you add Payment gateway Xendit ?
https://www.xendit.co/id/
in app

**Comment by byndrha** [comment]
supported already?

**Comment by HanielQC** [comment]
https://youtu.be/11WXkUCmFKY - Xendit payment is supported by the latest FluxStore 3.10.0 - Thank you and wish you all the best @all
